packages/hash/api/package.json

```diff
@@ -23,6 +23,7 @@
     "@aws-sdk/s3-presigned-post": "3.38.0",
     "@aws-sdk/s3-request-presigner": "3.38.0",
     "@hashintel/hash-backend-utils": "0.0.0",
+    "@hashintel/hash-graph-client": "0.0.0",
```

Comment:
I tried this package out in the API, and I'm wondering why this generated SDK exports multiple APIs which have to be initialized. I expected there to be something like a `GraphApi` which I can initialise and call methods on. Instead there are individual APIs for every "thing" we are storing in the API. For example for entities:

```TypeScript
import { EntityApi } from "@hashintel/hash-graph-client";

const entityApi = new EntityApi();

const entity = await entityApi.createEntity({
  account_id: "...",
  entity: { ... },
  ...
});
```

I think we should try and find a way to expose the methods on a single API. It would make the client easier to use in the HASH workspace, and be more consistent with how the Kratos SDK appears to function.

Reply:
I think this is done based on the tags we set up in the schema. Do you find it unergonomic to use when it's split up like this?

Reply:
My instinct is yes, it would be unergonomic. If I'm understanding correctly we'd have to do:
```TypeScript
import { EntityApi, LinkApi } from "@hashintel/hash-graph-client";

const entityApi = new EntityApi({ basePath: "http://graph-api.com" });

const entity = await entityApi.createEntity({ ... });

const linkApi = new LinkApi({ basePath: "http://graph-api.com" });

const link = await linkApi.createLink({ ... });
```
Whereas with a unified API we can reuse the configuration:

```TypeScript
import { GraphApi } from "@hashintel/hash-graph-client";

const graphApi = new GraphApi({ basePath: "http://graph-api.com" });

const entity = await graphApi.createEntity({ ... });

const link = await graphApi.createLink({ ... });
```

Reply:
Oh yeah the configuration needs to be explicitly shared in that case. I agree on this. I will see if i can keep the tags and add a unified tag called `Graph` :)

Reply:
Curious to get thoughts from other, perhaps there's a reason to split the graph into multiple APIs that I'm not thinking of (cc @Alfred-Mountfield @TimDiekmann )

Reply:
Thanks @thehabbos007 👍

Reply:
Tell me if you like this better f0918c77195c73d7422d1777cdb0edd13dab14e7
It does make it a fair bit simpler :)

Reply:
Great, that's giving me a single `GraphApi` which appears to have all the methods. The previous Apis are still exposed though, is it possible to easily remove these from the package? (i.e. remove `LinkApi`, `EntityApi`, etc)

Reply:
Ah didn't notice that. It would, but that means removing our tags entirely. I think we want to see what @Alfred-Mountfield and @TimDiekmann think about that. I guess it doesn't hurt to have the individual apis as well

Reply:
Yeah, doesn't hurt as we are the only ones that are going to use this SDK for a while anyway. If down the line it gets used by others publicly, might be worth removing the redundant API clients from the export to avoid confusion of what should be initialised.

Reply:
As we want different stores in the Graph (`DataTypeStore`, `EntityStore` etc.) I think it also makes sense to have multiple client APIs. Not a strong opinion, though.

Reply:
I think our 'different stores' in the Graph is more of an implementation detail, partly for code layout but also to allow us to have different storage mechanisms and such later on. 

The Graph Query Layer itself though should be a unified API, that's how we've been envisioning it anyway (e.g. if you were to do `get(Person)` it should return all `Person` entities across all storage solutions).

I think the solution in https://github.com/hashintel/hash/commit/f0918c77195c73d7422d1777cdb0edd13dab14e7 is good enough, and we shouldn't worry about getting rid of the individual APIs.